File: china_finance/china_finance/report/china_purchase_document_chain/china_purchase_document_chain.js
frappe.query_reports["China Purchase Document Chain"] = {
	onload(report) {
		if (!report.get_filter_value("company")) {
			report.set_filter_value("company", window.china_finance?.company_context?.default_company() || frappe.defaults.get_user_default("Company"));
		}
	},
};

File: china_finance/china_finance/report/china_purchase_document_chain/china_purchase_document_chain.py
import frappe

from china_finance.china_finance.report.china_purchase_reconciliation.china_purchase_reconciliation import (
	execute as execute_purchase_reconciliation,
)


def execute(filters=None):
	filters = frappe._dict(filters or {})
	filters.view_mode = "Purchase Order"
	return execute_purchase_reconciliation(filters)
